client-ts/signalr/src/HubConnection.ts

```diff
@@ -267,6 +267,9 @@ export class HubConnection {
         const removeIdx = handlers.indexOf(method);
         if (removeIdx !== -1) {
             handlers.splice(removeIdx, 1);
+            if(handles.length === 0) {
+                handles.delete(methodName);
```

Comment:
Isn't handlers an array? What does this do?

Reply:
Fixed.